media/css/protocol/components/_fxa.scss

```diff
@@ -0,0 +1,27 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+@import "../../../node_modules/@mozilla-protocol/core/protocol/css/includes/lib";
+
+// Hide elements marked for specific variations by default
```

Comment:
Is there a reason this should be under `/protocol/`? I see there's one in the `/pebbles/` directory already, but these are really just state classes and nothing directly related to either library. Would it make sense for this to live in `/css/base/` where any page can use it?

Reply:
Good idea. I don't think we should make that change in this PR though since it will conflict with #6173

Reply:
I don't see this file used in #6173, but I may be misunderstanding something?

Reply:
Ah you're right, that's a Pebbles page so it's using the Pebbles version. But I really meant I didn't want to move it to /base in this PR, since we'd move the Pebbles one to base as well.

Reply:
Yeah no worries, I filed an issue so we can move both at the same time, which probably makes more sense anyway https://github.com/mozilla/bedrock/issues/6211